### ISSUE TITLE
docker issues in all machines

### ISSUE THREAD

**Issue by abilashks** [docker/docker#12689]
We are facing issue in docker suddenly in all our machines.
docker ps and images command is taking more time and
docker run command is getting hanged always. any of you faced this issue. please help
We have rebooted and reinstalled docker also but that doesn't solve the issue. Is anything related to network?

sudo docker version
Client version: 1.5.0-dev
Client API version: 1.18
Go version (client): go1.3.3
Git commit (client): fc0329b/1.5.0
OS/Arch (client): linux/amd64
Server version: 1.5.0-dev
Server API version: 1.18
Go version (server): go1.3.3
Git commit (server): fc0329b/1.5.0
OS/Arch (server): linux/amd64

**Comment by unclejack** [comment]
@abilashks You're running a custom built Docker binary. Please update to Docker 1.6.0. Please provide full output of `docker info`, `docker version' and `uname -a` if you're still running into this problem.

I'm going to close this issue now as this issue needs exact steps on how to reproduce the problem and official binaries should be used.

Please comment on this issue. The issue will be re-opened if the required bits of information are provided.

**Comment by abilashks** [comment]
only in that machine we used custom docker rest of the machines are in 1.5.0 version.

once we cleared the /var/lib/docker folder and restarted the machine and docker service it worked. but this happened in mutiple machine any idea?

Client version: 1.5.0
Client API version: 1.17
Go version (client): go1.3.3
Git commit (client): a8a31ef/1.5.0
OS/Arch (client): linux/amd64
Server version: 1.5.0
Server API version: 1.17
Go version (server): go1.3.3
Git commit (server): a8a31ef/1.5.0

uname -a 
Linux integ-ci 2.6.32-431.20.3.el6.x86_64 #1 SMP Thu Jun 19 21:14:45 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux
Containers: 15
Images: 150
Storage Driver: devicemapper
 Pool Name: docker-8:5-28443042-pool
 Pool Blocksize: 65.54 kB
 Backing Filesystem: extfs
 Data file: /dev/loop0
 Metadata file: /dev/loop1
 Data Space Used: 13.27 GB
 Data Space Total: 107.4 GB
 Metadata Space Used: 18.59 MB
 Metadata Space Total: 2.147 GB
 Udev Sync Supported: true
 Data loop file: /var/lib/docker/devicemapper/devicemapper/data
 Metadata loop file: /var/lib/docker/devicemapper/devicemapper/metadata
 Library Version: 1.02.89-RHEL6 (2014-09-01)
Execution Driver: native-0.2
Kernel Version: 2.6.32-431.20.3.el6.x86_64
Operating System: <unknown>
CPUs: 48
Total Memory: 62.87 GiB
Name: integ-ci
ID: QRXA:OHBN:3B4N:J6V5:AGUF:3BWB:SW7W:BNNV:IJHA:JNAZ:26NB:NMFX
Username: integ
Registry: [https://index.docker.io/v1/]
